internal/devconfig/packages.go

```diff
@@ -143,13 +226,45 @@ func NewPackage(name string, values map[string]any) Package {
 		version = ""
 	}
 
+	var platforms []string
+	if p, ok := values["platforms"]; ok {
+		platforms = p.([]string)
+	}
+	var excludedPlatforms []string
+	if e, ok := values["excluded_platforms"]; ok {
+		excludedPlatforms = e.([]string)
+	}
+
 	return Package{
-		kind:    regular,
-		name:    name,
-		Version: version.(string),
+		kind:              regular,
+		name:              name,
+		Version:           version.(string),
+		Platforms:         platforms,
+		ExcludedPlatforms: excludedPlatforms,
 	}
 }
 
+// enabledOnPlatform returns whether the package is enabled on the given platform.
+// If the package has a list of platforms, it is enabled only on those platforms.
+// If the package has a list of excluded platforms, it is enabled on all platforms
+// except those.
+func (p *Package) enabledOnPlatform(platform string) bool {
```

Comment:
can this just be part of `IsEnabledOnPlatform` ?

Reply:
lol, yes